### DISCUSSION TITLE
Points

### DISCUSSION THREAD

**Discussion by ohnegesetz** [IBAX-io/go-ibax#19] [Rewards]
Hey! I created my wallet, but the additional 1500 pieces were not reflected in my account. And I invited about 5 friends of mine. What should my friends do for me to earn more points? Up together!

**Comment by mucar** [comment]
Yes, it seems there is a bug. Me and my wife experienced with same situation. 

We performed all the tweet tasks and get 8500 IBAX points. After that we created weaver wallet, but additional 1500 points didn't appear on the page http://ibax.io/user.

On the other hand weaver wallet page (https://weaver.ibax.io/browse) shows correct balance as 10.000.

My wallet address: 1168-2339-9243-7448-6536
My wife's wallet address: 1134-8785-0969-4324-7643

**Reply by ohnegesetz** [reply]
Yeah same bug. 
My wallet address : 0388-2965-0130-5408-2967

**Reply by HalloKyle** [reply]
Could you print a screen to show the issue？

**Reply by ohnegesetz** [reply]
Of course, I have different puns on these two pages:
![image](https://user-images.githubusercontent.com/97051458/148082030-d6860fe7-c397-4395-bdbc-b24f0c90b91a.png)
9999 points here.
![image](https://user-images.githubusercontent.com/97051458/148082078-0db57090-fd3e-492e-93b8-ff8dcf0ff953.png)
8500 here.
Up together!

**Reply by mucar** [reply]
I think this bug is the most important issue around here.

**Reply by HalloKyle** [reply]
That's not a bug that's the points you earned, and the wallet picture shows you have 9999 IBXC that's test coins for developers they can use that to build a ecosystem. After the airdrop event ends you can redeem the IBAX points to IBXC coins on the Mainnet work.

**Comment by Emirxt** [comment]
AİRDROP BİTTİKTEN SONRA İBAXLERİNİZ CÜZDANINIZDA GÖZÜKECEKTİR BU TESTNET SAYISI. CÜZDAN KODLARINIZI BİRYERE YAZMAYI UNUTMAYIN.
